Documentation/AdvancedBuildInstructions.md

```diff
@@ -250,3 +251,26 @@ Some OS distributions don't ship bleeding-edge clang-format binaries. Below are
 1. If you have a Debian-based (apt-based) distribution, use the [LLVM apt repositories](https://apt.llvm.org) to install the latest release of clang-format.
 2. Compile the SerenityOS-patched LLVM from source using `Toolchain/BuildClang.sh` as described above and use the compiled `Toolchain/Local/clang/bin/clang-format` binary in your editor and terminal. The meta-lint-ci pre-commit hook will automatically pick up the Toolchain clang-format binary.
 3. Compile LLVM from source as described in the LLVM documentation [here](https://llvm.org/docs/GettingStarted.html#compiling-the-llvm-suite-source-code).
+
+## Coverage builds
+
+For serenity builds, Meta/analyze-qemu-coverage.sh can help you.
+
+For lagom builds, you currently have to do the same thing manually:
+
+1. Enable `ENABLE_LAGOM_COVERAGE_COLLECTION` in BUild/lagom/CMakeCache.txt
+2. Build: `Meta/serenity.sh build lagom TestImageDecoder`
+3. Run tests: `(cd Tests/LibGfx; ../../Build/lagom/bin/TestImageDecoder)`
+4. Generate report, requires an LLVM checkout and build:
```

Comment:
Do we need a custom LLVM build? (That's how it sounds here)
Building with coverage already requires `clang` and I think it usually comes with `llvm-tools` which contains `llvm-profdata` and `llvm-cov`.